openapi/openapi.yaml

```diff
@@ -551,6 +578,25 @@ components:
       required:
         - batch_size
         - parameters
+    InspectionListingResponse:
+      type: object
+      description: Get listing of available inspections.
+      additionalProperties: false
+      properties:
+        inspections:
+          type: array
+          description: >-
+            A listing of inspections available.
+          items:
+            type: string
+            description: An inspection identifier (inspection id).
+            example: inspect-ABCXYZ
```

Comment:
`inspection-ABCXYZ`? just to be compliant with name convention we have :)

Reply:
Nice catch! Adjusted also in other parts :+1: